contributing_to_docs/doc_guidelines.adoc

```diff
@@ -715,6 +715,43 @@ This renders as:
 
 > Use the `GET` operation to do x.
 
+=== Error messages
```

Comment:
Note: @vikram-redhat suggested making this more generally "message text" rather than error-specific. I think that's a good idea as well.